src/Emitter.js

```diff
@@ -745,12 +756,39 @@
 	});
 
 	/**
+	 * If particles should be emitted during update() calls. Setting this to false
+	 * stops new particles from being created, but allows existing ones to die out.
+	 * @property {Boolean} emit
+	 */
+	Object.defineProperty(p, "autoUpdate",
+	{
+		get: function() { return this._autoUpdate; },
+		set: function(value)
+		{
+			if (this._autoUpdate && value === false)
+			{
+				ticker.remove(this.update, this);
+			}
+			else if (!this._autoUpdate && value === true)
```

Comment:
Given that you are converting to a boolean value when setting `_autoUpdate` at the bottom of this, it seems odd to use strict equality rather than just using `value` as falsy and truthy.